cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -528,16 +537,18 @@ class OverlappingMountDirValidator(Validator):
     """
     Mount dir validator.
 
-    Verify if there are overlapping mount dirs between shared storage and ephemeral volumes.
+    Verify if there are overlap mount dirs.
+    1. Shared storage directories can not overlap with each other.
+    2. Shared storage directories can not overlap with ephemeral storage directories.
+    3. Ephemeral storage directories can overlap with each other, because they are local to compute nodes.
     Two mount dirs are overlapped if one is contained into the other.
     """
 
-    def _validate(self, mount_dir_list):
-        overlapping_mount_dirs = _find_overlapping_paths(mount_dir_list)
+    def _validate(self, shared_mount_dir_list, local_mount_dir_list):
```

Comment:
I'd say we can avoid the two parameters here too, given that we're not making any difference in the printed message, so I'd keep the validator as it was before and call it with a single list: `shared_mount_dir_list + local_mount_dir_list`

Reply:
See comment above